04LessonFour/LessonFour.js

```diff
@@ -2,38 +2,48 @@ module.exports = function() {
 
   this.arrayPop = function (arr) {
     // should remove the last item in the array, then return the arr
-
+  arr.pop();
+  return arr;
   }
 
   this.arrayPush = function (arr, item1, item2, item3) {
     // should add these items onto the end of the array, then return the array
-
+    arr.push(item1, item2, item3);
+    return arr;
   }
 
   this.arrayShift = function (arr) {
     // should remove the first item in the array, then return the array
-
+    arr.shift([0]);
```

Comment:
Take a quick look [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/shift) to determine if the `[0]` is required for `.shift`